brewchat/src/main/java/com/example/brewchat/events/UserSignedUpEvent.java

```diff
@@ -4,4 +4,19 @@
  * Created by Ryanm14 on 4/25/2015.
  */
 public class UserSignedUpEvent {
+    private String username;
+
+    public UserSignedUpEvent(String username){
+        this.username = username;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
```

Comment:
I would delete this method. I think that events should be inmutable once they are created.